src/libcpu/src/jit/jit_float.cpp

```diff
@@ -56,155 +56,112 @@ absXmmSd(PPCEmuAssembler& a, const PPCEmuAssembler::XmmRegister& reg)
    a.pand(reg, maskXmm);
 }
 
-template <bool ShouldTruncate>
```

Comment:
As before, please indicate which Sd instruction is corrupting the high-order double of the Xmm.